app/tests/test_urls.py

```diff
@@ -5,10 +5,22 @@
 class test_accounts_page(TestCase):
     def test_getLogin(self):
         client = Client()
-        response = client.get("http://127.0.0.1:8000/accounts/login/")
+        response = self.client.get("http://127.0.0.1:8000/accounts/login/")
         assert response.status_code == 200
 
     def test_landing_redirect(self):
         client = Client()
-        response = client.get("http://127.0.0.1:8000", follow=True)
+        response = self.client.get("http://127.0.0.1:8000", follow=True)
         SimpleTestCase().assertRedirects(response, "/accounts/login/?next=/")
+
+
+class test_signup_page(TestCase):
+    def test_get_signup(self):
+        client = Client()
+        response = client.get("http://127.0.0.1:8000/accounts/signup/")
+        assert response.status_code == 200
+
+    def test_get_signup_by_name(self):
+        client = Client()
+        response = client.get(reverse("signup"))
```

Comment:
So actually pressing the button on the login page cannot be unit tested, because that would require using Selenium or equivalent, which would be too bulky for a unit test. This calls the same url pattern name, effectively doing the same thing.

Reply:
That should probably be fine, if we had more time for this project I'd say lets to system tests with Selenium, but I think this will be fine.